jumpscale/tools/nginx/nginxserver.py

```diff
@@ -0,0 +1,30 @@
+from jumpscale.god import j
+from jumpscale.core.base import Base, fields
+
+
+class NginxServer(Base):
+    name = fields.String(default="main")
+    path = fields.String(default="~/sandbox/cfg/nginx/main/nginx.conf")
+
+    def install(self):
+        j.sals.process.execute("apt install nginx", showout=True, die=False)
```

Comment:
You need to add `-y` or else this won't work

Reply:
never use apt install anything, if you must use apt
use apt-get install -y....

Reply:
https://manpages.debian.org/jessie/apt/apt.8.en.html#SCRIPT_USAGE